dependencies.yaml

```diff
@@ -60,7 +60,7 @@ libcurl:
 # and would otherwise be overwritten by the version used there
 libevse-security:
   git: https://github.com/EVerest/libevse-security.git
-  git_tag: v0.8.0
+  git_tag: 5c0b6655a49167e8140f08e155fa43c823134897
```

Comment:
will need to be changed before merge

Reply:
I have a generic question for this. Do we really need to make a version for each small version release that we have? Libocpp is also a hash: https://github.com/EVerest/everest-core/blob/b11c690bc2158b713a16c5e81be73341a94cfd74/dependencies.yaml#L67

Reply:
Since libocpp updates relatively frequently we only tag close to releases. But I think libevse-security doesn't update quite as often, so I would probably tag it. But this probably would need a version bump in the libevse-security CMakeLists.txt anyway in an additional commit